doc/sphinx/notifications/index.rst

```diff
@@ -0,0 +1,79 @@
+.. _notification_templates:
+
+######################
+Notification Templates
+######################
+
+Brand administators can configure the notifications sent by IvozProvider:
+
+- Email sent when a new voicemail is received
+
+- Email sent when a new fax is received
+
+.. hint:: When no custom notification is configured, default ones will be used
+
+Notifications are created in two steps: Create a notification type and add contents to the notification for each
+required language.
+
+***************************
+Creating a new notification
+***************************
+
+Brand administrators can create new notification templates in **Brand configuration** > **Notification templates**:
+
+Fields are nearly self-explanatory:
+
+.. glossary::
+
+    Name
+        Used to identificate this notification template
+
+    Type
+        Determine the notification type. Each notification type has its own substitution variables avaiable to replace
+        the contents of the subject and body.
+
+****************************
+Adding Notification contents
+****************************
+
+Once the notification has been created, you can add different language contents. IvozProvider will automatically use
+the correct language based on the destination:
```

Comment:
proper?

Reply:
This is the same string as previous one but in translation file.